application/include/mmul.hpp

```diff
@@ -37,16 +38,21 @@ void mmul(Executor ex, MatrixXd &a, MatrixXd &b, MatrixXd &c) {
       }
     }
   };
-  ex.bulk_execute(mul, 1, a, b, c);
+  ex.bulk_execute(std::bind(mul, std::ref(a), std::ref(b), std::ref(c),
```

Comment:
Why not capture `a`, `b`, `c` in the lambda itself?

Reply:
Fixed